src/main/kotlin/no/nav/syfo/brev/behandler/BehandlerVarselService.kt

```diff
@@ -0,0 +1,60 @@
+package no.nav.syfo.brev.behandler
+
+import no.nav.syfo.dialogmote.domain.*
+import no.nav.syfo.domain.PersonIdentNumber
+import java.util.UUID
+
+class BehandlerVarselService(
+    private val behandlerDialogmeldingProducer: BehandlerDialogmeldingProducer,
+) {
+    fun sendVarsel(
+        behandlerRef: String,
+        arbeidstakerPersonIdent: PersonIdentNumber,
+        document: List<DocumentComponentDTO>,
+        pdf: ByteArray,
+        varseltype: MotedeltakerVarselType,
+        varselUuid: UUID,
+        varselParentUuid: UUID?,
+    ) {
+        behandlerDialogmeldingProducer.sendDialogmelding(
+            dialogmelding = KafkaBehandlerDialogmeldingDTO(
+                behandlerRef = behandlerRef,
+                personIdent = arbeidstakerPersonIdent.value,
+                dialogmeldingUuid = varselUuid.toString(),
+                dialogmeldingRefParent = varselParentUuid?.toString(),
+                dialogmeldingRefConversation = getConversationUuid(varselUuid, varselParentUuid).toString(),
+                dialogmeldingType = getDialogMeldingType(varseltype),
+                dialogmeldingKode = getDialogMeldingKode(varseltype),
+                dialogmeldingTekst = document.serialize(),
+                dialogmeldingVedlegg = pdf,
+            )
+        )
+    }
+
+    private fun getConversationUuid(varselUuid: UUID, varselParentUuid: UUID?): UUID {
+        return varselParentUuid ?: varselUuid
+    }
```

Comment:
Dette ser ut som fungerer når det er opptil 2 "runder" i en "convertsation".
Mulig jeg ser noe feil her, men for meg ser det ut som far(parent) og bestefar(conversation) vil være lik ved 3 "runder".

Eksempel for slik jeg forstår at det skal være:
1. Innkalling: Uuid = conversation, parent=null
2. Endring tid og sted 1: uuid er uuid, parent = conversation = pkt1 sin uuid
3. Endring tid og sted 2. uuid er uuid, parent er pkt2 sin uuid, og conversation er pkt 1 sin uuid //  Om jeg leser koden riktig blir det feil her på pkt 3. fordi parent blit satt til å være sammen som conversation.

Reply:
Ja, kan se om parent-referansen kan forbedres. 

conversation tror jeg er riktig: "Det anbefales at den er identisk med
MsgInfo/MsgId for første melding i en dialog"

Reply:
Det jeg mener er fei er Conversation, ikke parent. Ref det du selv skrev ovenfor, så ser det feil ut når det blir mer enn 2 meldinger i en conversation.

Reply:
Ja, enig i det: så det er innkalling-varsel-uuid som skal være "conversation". Men det er faktisk det koden gjør nå siden parent peker til innkallingen. 

Men i alle fall skal jeg fikse litt og forsøke å gjøre det klarere.